### ISSUE TITLE
Collaborating on upcoming Tent Release

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/editorial#49]
Jan recently wrote about Hoodie’s [The (Larger) Hoodie Roadmap](https://github.com/hoodiehq/discussion/issues/88)

tl;dr The larger milestones are:

1. Contributors
2. Developers
3. Users

I would love to put our heads together and think about how we can improve as the Hoodie community to become more friendly towards new and existing contributors.

The motivation is this: from the code side of Hoodie, we worked very hard in the past year to lower the barrier as much as possible to become a new code contributor. The way we did it is to split up the code into [a lot of separate modules](https://github.com/hoodiehq/hoodie#why-is-there-no-code-in-this-repository), which are all well tested & documented, and interested people can pick issues for one of these modules and work on them, without the necessity to understand how all of Hoodie works. For example, https://github.com/hoodiehq/hoodie-client-log is a little library for easy logging to the browser console. You can use it by itself, it’s self-contained, it just happens to be used by the Hoodie client. And because it’s only a few files with a few lines of code, it’s easy to grasp and to contribute to. And all bugs fixed and features added to it – Hoodie benefits from it automagically. 

So.

Hoodie is in a great state to scale in terms of code contributors (and other contributors of course!). Maybe the editorial team has some good ideas how we could use writing / twitter / you name it to try achive the same goal: finding new contributors and become more friendly for contributors in general?

For example, we could write a post for first-time open source contributors, explaining things like `git`, Github, forking, pull requests, Node etc, that would be helpful for many (JavaScript) open source projects out there, and we could link to the blog post from our starter issues. 

Maybe we could show the starter issues right on the http://hood.ie landing page? I’m sure we come up with more ideas once we thing more about it.

Hope this all makes sense :) I’m curious to hear what you think

**Comment by Charlotteis** [comment]
If we had a page on the website showing starter issues, that would be amazing. How do you imagine that working? A link to the ubersicht or do you want to build a page using the GitHub API, falling back to the ubersicht link if it doesn't work?

I'd love to have an explanation on git, github, forking, PR's & node etc.
How about we create a little guide we can host on its own website, writing in a way that is not specific to Hoodie, but people all over the community can link to? Basically I have wanted to write some of those things for some time, it feels like having a standalone website would help a lot of people in the industry.

**Comment by gr2m** [comment]
I would say that for the upcoming release, it would make sense to directly load & embed starter issues on the hood.ie landing page, and link to ubersicht for more. 

For the explanation  on basics, I’d suggest we do a Hoodie specific first, and then see if we can make a generic one together with other projects? We could make a blog post for Hoodie that we can link to for a start, and then see how we can build upon it?

**Comment by Charlotteis** [comment]
> I would say that for the upcoming release, it would make sense to directly load & embed starter issues on the hood.ie landing page, and link to ubersicht for more.

:+1: 

> For the explanation on basics, I’d suggest we do a Hoodie specific first, and then see if we can make a generic one together with other projects? We could make a blog post for Hoodie that we can link to for a start, and then see how we can build upon it?

:+1: ok

**Comment by gr2m** [comment]
Another idea I had for the Tent release would be a landing page for new and existing contributors. It would be like docs.hood.ie, but documentation for contributors, not users. We could document conventions like how we do READMEs, how we prepare issues to make them "starter" issues, what tools we use, why, and how, we could make tutorials where we go through the code base explaining how things work together. And all this for both, code & editorial contributors, of course. I’d love to work on this, and I’ll have time in February / March, but I will need help with thinking this through and managing what needs to be done and by whom.

Anyone interested? This would be a time commitment of a few hours per week, for February, maybe March.

**Comment by nickcolley** [comment]
I should be available to help with getting anything into the website. :+1:

**Comment by ianstalter123** [comment]
I'd be interested in seeing a dumbed down overview/tutorial/ of how the hapi routes and authentication are laid out for reference etc

**Comment by jsimplicio** [comment]
@gr2m hey the last thing you said here should be referenced on the new thread about the landing page. It pretty much covers how it should be layout

**Comment by gr2m** [comment]
done https://github.com/hoodiehq/discussion/issues/90#issuecomment-185015986 :+1:

**Comment by renrutnnej** [comment]
Created an editorial issue https://github.com/hoodiehq/editorial/issues/61 for the /contributors request in this issue :+1:
